SPECS/maven.spec

```diff
@@ -0,0 +1,55 @@
+###########################
+# Set global SPEC variables
+###########################
+
+%global prefix 	/usr/local
+
+Name:		maven
+Version:	3.3.9
+Release:	1%{?dist}
+Summary:	Apache Maven is a software project management and comprehension tool.
+
+Group:		Development/Build Tools
+License:	Apache Software License
+URL:		https://maven.apache.org/index.html
+Source:		apache-maven-%{version}-bin.tar.gz
+
```

Comment:
Please add a `Requires: java` line here, since maven will not run without Java installed.  This will cause Yum to install Java if it is not present, which is the normal way RPM dependencies are resolved.